includes/definitions/discovery/jetstream.yaml

```diff
@@ -1,4 +1,4 @@
-mib: TPLINK-SYSMONITOR-MIB
+mib: TPLINK-MIB:TPLINK-PRODUCTS-MIB:TPLINK-SYSMONITOR-MIB
```

Comment:
Since you aren't adding more sensors, are these extra mibs needed?

Reply:
I would like to add sensors, but without working MIBs to work from, I'm stuck.

Reply:
It's fine to add the mibs, but there is no need to import them yet.  If you are yet going to add sensors here, go ahead.  I'm not sure how much those particular mibs will help ;)